appveyor.yml

```diff
@@ -173,4 +173,9 @@ on_success:
 
 
 on_finish:
-  - ps: 'if (Test-Path ".\junit-results.xml") { (new-object net.webclient).UploadFile("https://ci.appveyor.com/api/testresults/junit/$($env:APPVEYOR_JOB_ID)", (Resolve-Path .\junit-results.xml)) }'
+  - ps: |
+      (new-object net.webclient).UploadFile(
+        "https://ci.appveyor.com/api/testresults/junit/$($env:APPVEYOR_JOB_ID)",
+        (Resolve-Path .\junit-results.xml)
+      )
+      $LastExitCode = 0
```

Comment:
I haven't seen a cache upload failure fail the build. In fact, if you look [here](https://ci.appveyor.com/project/xoviat/pyinstaller/build/1.0.9/job/14i71eaqa4itto0i), you can see the cache failed but the build passed.